### ISSUE TITLE
Exception is get thrown while passing data to mini_line_chart

### ISSUE THREAD

**Issue by kingdj** [elanandkumar/covid19info-IN#1]
Hey I tried to supply my data to ur mini_line_chart but throws exception that 'type int is not a sub type string in type cast...'
Can you pls rectify..

Even Ur each JSON data is of string type while in mycase some of the values are of integer.. isnt is the cause.
If so thn what can be done.

**Comment by elanandkumar** [comment]
Hello @kingdj,
Could you please provide more details about the error you are getting. May be stack trace? To help you, this chart is customized to my need where data is coming as String. So, feel free fork this repo and do the modification as per your need and you can create Pull request too.

In case you had mixed data, you can do a type conversion before feeding the data to chart or you can modify the component accordingly.

Hope this helps.

**Comment by kingdj** [comment]
Yes I did type conversion but it's a mess as if values doesn't get counted then either catch is caught & flutter container gets red with error.. which hopefully nt good for citizens to see or if gets skipped then graph is get plotted wrong..

**Comment by kingdj** [comment]
Hey btw can I get info on how live JSON API link with domain name can be generated.. what's the platform used.. is it a ReST API.?.

Started android after a long time, many new things has cameup.. so trying to understand it all.
